test/fw/bin/pbs_snapshot

```diff
@@ -140,6 +127,7 @@ def childsnap_thread(logger, host, acct_logs, daemon_logs, map_file, anonymize,
 
     # Create a directory on the remote host with a unique name
     # We will create the snapshot here
+    timestamp = str(int(time.time()))
```

Comment:
This can potentially result into every thread getting a different timestamp. Is that ok?

Reply:
yes that's fine, the timestamp is only used to name the snapshots uniquely on the remote hosts to avoid name conflicts in case we capture another snapshot on that host in the future. Afterwards, the snapshots from the remote hosts are renamed as "<hostname>_snapshot.tgz" when we copy them over to the main snapshot.